MAUI/Calendar/customizations.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: post
+title: Customizations in MAUI Calendar widget | Syncfusion
+description: Learn here all about Customizations features of Syncfusion .NET MAUI Calendar (SfCalendar) widget and more. 
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Customizations in MAUI Calendar (SfCalendar)
+Each cell of the Month, Year, Decade and Century can be customized in MAUI Calendar (SfCalendar).
+
+## Month cell customization
+You can customize the calendar month view cell by using the `MonthView` property of `SfCalendar`.
+
+*    **Month dates** – You can customize the month date's text style and background of the `Calendar` by using the `TextStyle` and `Background` properties of `MonthView`.
+
+*    **Today date** – You can customize the today date text style and background of the `Calendar` by using the `TodayTextStyle` and `TodayBackground` properties of `MonthView`.
+
+*    **Trailing and Leading dates** – You can hide the trailing and leading date by using the `ShowTrailingAndLeadingDates` property in the `CalendarMonthView` class. You can also customize the Trailing and Leading dates text style and background of the `Calendar` by using the `TrailingLeadingDatesTextStyle` and `TrailingLeadingDatesBackground` properties of `MonthView`.
```

Comment:
You can hide the trailing and leading dates by using the `ShowTrailingAndLeadingDates` property in the `CalendarMonthView` class. You can also customize the trailing and leading dates text style and background of the `Calendar` by using the `TrailingLeadingDatesTextStyle` and `TrailingLeadingDatesBackground` properties of `MonthView.`

Reply:
Content changed